src/RestApi/StoreApi/Schemas/CartShippingRateSchema.php

```diff
@@ -79,7 +79,21 @@ protected function get_properties() {
 				'context'     => [ 'view', 'edit' ],
 				'readonly'    => true,
 				'items'       => [
-					'type' => 'string',
+					'type'       => 'object',
+					'properties' => [
+						'name'     => [
+							'description' => __( 'Name of the item.', 'woo-gutenberg-products-block' ),
+							'type'        => 'string',
+							'context'     => [ 'view', 'edit' ],
+							'readonly'    => true,
+						],
+						'quantity' => [
+							'description' => __( 'Quantity of the item in the current package.', 'woo-gutenberg-products-block' ),
+							'type'        => 'integer',
```

Comment:
Maybe use `number` here - whilst core uses int, it's filterable and could be floating point.

Reply:
TIL. Done in 7cc35255322d2882e7df54125c354e098f05c213.